qemu/tests/resize_raw_img.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
+
+from virttest import data_dir
+from virttest import error_context
+from virttest import utils_numeric
+from virttest.qemu_storage import QemuImg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    A 'qemu-img' resize test.
+
+    1.create a raw image
+    2.change the raw image size * n
+    3.verify resize * n
+
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _sum_size_changes(size_changes):
+        """
+        Sum the list of size changes.
+
+        :param size_changes: list of size changes
+        """
+        res = []
+        for change in size_changes:
+            s = int(utils_numeric.normalize_data_size(change, "B")
+                    ) * (-1 if '-' in change else 1)
+            res.append(s)
+        return sum(res)
+
+    def _verify_resize(img_size, expected_size):
+        logging.info("Verify the size of  %s is %s." %
+                     (img.image_filename, expected_size))
+        if img_size != expected_size:
+            test.fail("Got image virtual size: %s, should be: %s." %
+                      (img_size, expected_size))
+
+    def _resize(size_changes):
+        for idx, change in enumerate(size_changes):
```

Comment:
Name it `change` is weird, `size` is better to me. No strong opinion on this, up to you.

Reply:
updated, thanks